style.css

```diff
@@ -0,0 +1,102 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box; 
+}
+
+body{
+    background-color:floralwhite;
+    background-blend-mode:darken;
+    
+}
+
+.name{
+    color: maroon;
+}
+
+nav{
+    display: flex;
+    align-items: center;
+    padding-top: 2.5rem;
+    padding-left: 1rem ;
+    padding-right: 1rem;
+    justify-content: space-between;
+}
+
+ul li{
+    display:flex;
+    flex-direction: column;
+    list-style-type:none;
+    padding: 0.8rem 1.4rem;
+    font-size: smaller;
+}
+
+ul li a{
+    text-decoration: none;
+    color:darkgrey;
+    font-weight:bold;
+}
+
+ul li a:hover{
+    color: maroon;
+    transition: .4s;
+}
+
+.container2{
+    position: absolute;
+    top: 12rem;
+    left: 5rem;
+    width: 100%;
+    margin-top: 15rem;
+    padding: 10px 0px;
+}
+
+h1{
+    font-size: 4rem;
+    color:maroon;
+    margin: 20px 0px 20px;
+}
+
+h3{
+    color: darkgrey;
```

Comment:
Use black or any darker color so it will be more visible

Reply:
Okay .. Thanks